### ISSUE TITLE
Some questions on ViTTT.

### ISSUE THREAD

**Issue by ieellee** [LeapLabTHU/ViTTT#5]
@tian-qing001 Dear Dongchen:

I have a few questions while reading the ViTTT open-source code:

1. The batch size mentioned in the paper, is it referring to the image level or all the vision tokens within an image? I'm a bit confused because gradient descent is used.

2. In the current code, is the number of training epochs for the TTT layer hard-coded to `1`?

Thank you very much for your patience, and I look forward to your response.

**Comment by tian-qing001** [comment]
Hi @ieellee, 
1. TTT views $(K_i, V_i),i=1,\cdots,N$ as a mini dataset. When using full-batch gradient descent, the batch size equals to the number of tokens in a single image $N$.
2. Yes. In our experiments, multiple inner epochs show limited performance gains, while leading to throughput degradation and training instability (Table 2 of the paper).
